### PR TITLE
jeffrey-jakubowsky adds eagan file

### LINKED ISSUE #6: jeffrey-jakubowsky practices using GitHub Flow
- [x] Create a branch off of master
- [x] Add some commits
- [x] Open a pull request
- [x] Collaborate
- [x] Merge when we get at least 1 thumbs up

### PR DESCRIPTION
**Pull request by jeffrey-jakubowsky**
The purpose of this pull request to....

fix bug and ask a question.

This PR closes #6

### CODE REVIEW

File: jeffrey-jakubowsky-eagan.md [hunk 1]
```diff
@@ -1,8 +1,11 @@
 # Eagan, MN
 
 ### Schools
+Cedar Elementary
+Rahn Elementary
 
 ### Parks
-
+Blackhawk Park
+Turquoise Park
```
Comment (bill-kunze) [inline jeffrey-jakubowsky-eagan.md:11]: Where is a good place to eat?

File: jeffrey-jakubowsky-eagan.md [hunk 2]
```diff
@@ -1,8 +1,11 @@
 # Eagan, MN
 
 ### Schools
+Cedar Elementary
+Rahn Elementary
```
Comment (bill-kunze) [inline jeffrey-jakubowsky-eagan.md:6]: Do the kids get to go to High School?

**Comment by jeffrey-jakubowsky** [comment]
@bill-kunze  does this look better?

**Review by bill-kunze — APPROVED** [review]
Looking good.

**Pull request merged**